File: Python/1015.py
# -*- coding: utf-8 -*-
import math

x1, y1 = input().split(" ")
x2, y2 = input().split(" ")

x1 = float(x1)
y1 = float(y1)
x2 = float(x2)
y2 = float(y2)
dist = math.sqrt(math.pow((x2 - x1), 2) + pow((y2 - y1), 2))
        
print("%.4f" % dist)

File: Java/1015.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);
        
        double x1 = leitor.nextDouble();
        double y1 = leitor.nextDouble();
        double x2 = leitor.nextDouble();
        double y2 = leitor.nextDouble();
        double dist = Math.sqrt(Math.pow((x2 - x1), 2) + Math.pow((y2 - y1), 2));
        
        System.out.printf("%.4f\n", dist);
    }
 
}